CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Unreleased
 
 - [...]
+- **[BREAKING CHANGE]** `Item`: `leftTitle` attribute can receive `string` or `node`
+- **[NEW]** Introducing `ItemBigData` component
```

Comment:
nit: it seems you also added few CSS fixes to ride axis and Messaging stuff, could you add that to the CHANGELOG please?

Reply:
- [ ] Update CHANGELOG